CoastalResilienceNetwork/GeositeFramework
# Ensure underscore is available through Use config

## Overview

Import underscore locally and add it to the Use config so that plugins
can reference it.

Plugins that don't directly import underscore reference it through
Use, but it is properly made available through that interface.
Because of that, require.js tries to find it in the ESRI/DOJO package
and can't, which causes underscore to be undefined.

Connects to #1043 
Connects to #669

## Testing Instructions

- Before switching to this branch, bring this plugin into your framework: https://github.com/CoastalResilienceNetwork/ecological-effects-slc and temporarily remove all other plugins.
- Load up the framework. You should see the error reported in #669.
- Now clone this branch and restart the dev server.
- Reload the site, the error should be gone and the plugin should load.
- Try adding in other plugins, and ensure things still work.
